src/docs/ganache/reference/cli-options.md

```diff
@@ -988,3 +988,68 @@ The aliases `--host` and `--hostname` have been deprecated, use `-h` instead.
 The port to listen on. The default is `8545`.
 
 The alias `--port` has been deprecated, use `-p` instead.
+
+## Detach mode
+
+Ganache can be started as a background process by passing the `--detach` option, along with any valid combination of start-up options.
+
+**`--detach`**, **`-D`**, **`--😈`**
```

Comment:
Does `--😈` work on all supported OS?

Reply:
In order for it to show up correctly the terminal's font does need to support it. But the characters are parsed correctly. Thought I've been having my doubts on if we should ship with `😈`. Since this isn't actually a daemon